### PR TITLE
fix(general bugs): workflow fixes

### PR DESCRIPTION
**Pull request by amovfx**
-taro nodes connect in order when dropped into the graph
-Taro info asset draw title says 'No Assets' if there are no assets

### CODE REVIEW

**Review by jamaljsr — COMMENTED** [review]
For the Info sidebar, I could have been more specific. The problem I see with this is having a divider with no content below it looks weird. So I was thinking to add text below the divider stating "This node does not have any assets.", so there is content below it.

It looks like code coverage for the `network.ts` file dropped with this update.

**Comment by amovfx** [comment]
Thanks for that clarity. Sorry for missing that code coverage. Ill update this asap.

**Comment by amovfx** [comment]
```
 network.ts                           |   99.69 |      100 |   98.86 |     100 |
```
I haven't seen this before, I have 100% line coverage, why would statements and functions be less than 100?

**Comment by jamaljsr** [comment]
Most likely you have an `if` statement but the `else` condition is never hit. Open the `coverage/lcov-report/index.html` for more details on what's not covered.

**Comment by amovfx** [comment]
Super interesting how a nuanced a language can be. Looks like a sorted array apparently wont sort if it doesn't need to. Pretty cool optimization.

File: src/components/designer/taro/info/AssetsList.tsx [hunk 1]
```diff
@@ -10,6 +11,9 @@ import AssetInfoDrawer from './AssetInfoDrawer';
 
 const Styled = {
   Wrapper: styled.div``,
+  P: styled.p`
```
Comment (jamaljsr) [inline src/components/designer/taro/info/AssetsList.tsx:14]: This should have a more descriptive name, such as `Empty`.

**Review by jamaljsr — COMMENTED** [review]
tACK 👍

This looks good now. Just one small nitpick.

File: src/components/designer/taro/info/AssetsList.tsx (hunk 1 above)
Comment (amovfx) [reply src/components/designer/taro/info/AssetsList.tsx:14]: Done.

**Review by jamaljsr — APPROVED** [review]

**Pull request merged**